docs/python_support/reference.md

```diff
@@ -0,0 +1,771 @@
+# Python Nodes in OpenVINO Model Server {#ovms_docs_python_support_reference}
+
+## Introduction
+
+**This feature is currently in preview, meaning some behaviors of the feature and user interface may change in future versions**
+
+ Starting with version 2023.3, OpenVINO Model Server supports execution of custom Python code. Such code can execute simple pre- or post-processing as well as complex tasks like image or text generation. 
+ 
+ Python execution is enabled via [MediaPipe](../mediapipe.md) by the built-in `PythonExecutorCalculator` that allows creating graph nodes to execute Python code. Python nodes can be used as standalone servables (single node graphs) or be part of larger MediaPipe graphs.
+
+ Check out the [quickstart guide](quickstart.md) for a simple example that shows how to use this feature.
+
+ ## Building Docker Image
+
+The publicly available `openvino/model_server` image on Docker Hub supports Python, but does not come with external modules installed. If Python is all you need then you can use the public image without modification. Otherwise, you will need to extend the public image with additional layers that install any modules required for your Python code to run. For example, let's say your code requires numpy.
+In that case, your Dockerfile may look like this:
+
+```dockerfile
```

Comment:
we could add a reference to https://github.com/openvinotoolkit/model_server/blob/main/demos/python_demos/Dockerfile, edit requirements.txt and run `make python_image`

Reply:
I added reference below, but I also think that this sample is useful here. It explicitly says what needs to be done to prepare a custom image.